daprdocs/content/en/concepts/components-concept.md

```diff
@@ -137,3 +137,10 @@ Dapr allows custom [middleware]({{< ref "middleware.md" >}}) to be plugged into
 {{% alert title="Note" color="primary" %}} 
 Since pluggable components are not required to be written in Go, they follow a different implementation process than built-in Dapr components. For more information on developing built-in components, read [developing new components](https://github.com/dapr/components-contrib/blob/master/docs/developing-component.md).
 {{% /alert %}}
+
+### Conversation
+
+Dapr provides a way for developers to migrate and converse between different large language model servers and toolkits.
```

Comment:
`Dapr provides developers a way to abstract the interaction with large language models with built-in security and reliability features`